README.md

```diff
@@ -32,14 +44,16 @@ directly in CSS/HTML:
 ```
 
 By hooking into a breakpoint definition, `@artsy/react-responsive-media` takes
-this imperative approach and makes it declarative:
+this imperative approach and makes it declarative.
```

Comment:
The HTML/CSS version is also declarative, no? How about something like “fresnel brings this same declarative approach to responsive design into the React world”?